src/frontend/js/hooks/useTeacherPendingContractsCount/index.ts

```diff
@@ -13,30 +11,18 @@ const useTeacherPendingContractsCount = ({
   organizationId,
   courseProductRelationId,
 }: UseTeacherPendingContractsCountProps) => {
-  const {
-    item: training,
-    states: { isFetched: isTrainingFetched },
-  } = useCourseProductRelation(courseProductRelationId, {
-    organization_id: organizationId,
-  });
-
```

Comment:
The commit body does not explicitly explain the purpose of this chunk removal imo 🤔

Reply:
I think it does.

We had to retrieve the `courseProductRelation` in order to use `courseProductRelation.course.course_id` and `courseProductRelation.course.product_id` with `useOrganizationContracts`.
Now that `useOrganizationContracts` is usable with a `courseProductRelationId`, this request is no longer used.

I realise that i don't need to manually enable  `useOrganizationContracts`. I'll remove it.
`isActive` was there because had to wait `courseProductRelation` to be fetch before requesting contracts with product and course ids.

Reply:
Ok I got it 👌